website/redirects.txt

```diff
@@ -85,6 +85,7 @@
 /docs/commands/server-join.html               /docs/commands/server/join.html
 /docs/commands/server-members.html            /docs/commands/server/members.html
 /docs/runtime/schedulers.html                 /docs/schedulers.html
+/docs/internals/scheduling.html               /docs/internals/scheduling/scheduling.html
```

Comment:
I decided to move this to the document that resembles the old content, rather than the new index page that links to scheduling and preemption, but the other way is reasonable too.